pkg/controllers/job/state/running.go

```diff
@@ -91,6 +91,10 @@ func (ps *runningState) Execute(action v1alpha1.Action) error {
 				}
 				return true
 			}
+			if status.Pending == jobReplicas {
```

Comment:
should it be `status.Pending > jobReplicas - totalMinavailable`

Reply:
yes, you're right, it's more accurate, I'll fix it

Reply:
I have fixed it

Reply:
I have a confusion why totalMinavailable  can not be equal to Job.Spec.MinAvailable? #2895

Reply:
@xiefan-github 
When I first raised the question, I was thinking that if the number of pending pods is lower than minavailable, the pod should be in the pending state.